server/src/storage/uncompressed_data_manager.rs

```diff
@@ -59,19 +61,26 @@ pub(super) struct UncompressedDataManager {
     // TODO: This is a temporary field used to fix existing tests. Remove when configuration component is changed.
     /// If this is true, compress finished buffers directly instead of queueing them.
     compress_directly: bool,
+    /// Log of the used uncompressed memory in bytes, updated every time the used memory changes.
+    pub(super) used_uncompressed_memory_log: Log,
+    /// Log of the amount of ingested data points, updated every time a new batch of data is ingested.
+    pub(super) ingested_data_points_log: Log,
+    /// Log of the total used disk space in bytes, updated every time uncompressed data is spilled.
+    pub used_disk_space_log: Arc<RwLock<Log>>,
```

Comment:
As mention above maybe expand the comment for `used_disk_space_log` to state that it is shared for all components in the storage engine and thus must be synchronized.

Reply:
See [this](https://github.com/ModelarData/ModelarDB-RS/pull/63#discussion_r1071995640) comment.